app/src/main/res/layout/activity_main.xml

```diff
@@ -17,19 +17,43 @@
         android:layout_marginLeft="16dp"
         android:layout_marginRight="16dp"
         android:layout_height="wrap_content"
-        app:layout_constraintBottom_toTopOf="@+id/title"
+        app:layout_constraintBottom_toTopOf="@+id/character_name"
         android:text="@string/main_button"/>
 
     <TextView
-        android:id="@+id/title"
+        android:id="@+id/character_name"
         android:layout_width="wrap_content"
         android:layout_height="wrap_content"
-        android:text="@string/main_title"
+        android:text="@string/character_name"
         app:layout_constraintBottom_toBottomOf="parent"
         app:layout_constraintEnd_toEndOf="parent"
         app:layout_constraintStart_toStartOf="parent"
         app:layout_constraintTop_toTopOf="parent" />
 
+    <TextView
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:id="@+id/character_status"
+        android:text="@string/character_status"
+        app:layout_constraintTop_toBottomOf="@+id/character_name"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintEnd_toEndOf="parent"
+        android:layout_marginTop="16dp"
+
+        />
+
+    <ImageView
+        android:id="@+id/character_image"
+        android:layout_width="wrap_content"
```

Comment:
try to fit the image on the 20% of the screen width and to have a aspect ratio of 1:1